webpack.loader.js

```diff
@@ -0,0 +1,33 @@
+// Export from "./my-custom-loader.js" or whatever you want.
+
+const makepotPlugin = require( '@wordpress/babel-plugin-makepot' )();
+
+module.exports = require( 'babel-loader' ).custom( () => {
+	return {
+		// Passed Babel's 'PartialConfig' object.
+		config( cfg ) {
+			if ( -1 !== cfg.options.filename.indexOf( 'editor.js' ) ) {
```

Comment:
We'd likely need more specificity here but suppose this is just to the test if the approach works?